README.md

```diff
@@ -36,14 +36,23 @@ We are a new and growing project, and welcome [contributions](CONTRIBUTING.md).
 - [API Design Guide](API_DESIGN_GUIDE.md)
 - [Call for Contributions](https://github.com/keras-team/keras-nlp/issues?q=is%3Aissue+is%3Aopen+label%3A%22contributions+welcome%22)
 
-## Quick Start
+## Installation
 
-Install the latest release:
+To install the latest official release:
 
 ```
 pip install keras-nlp --upgrade
 ```
 
+To install the bleeding edge, we recommend using pip to install directly from
```

Comment:
"To install from source"? I want to say `nightlies` but obviously we don't have nightly builds.

Reply:
Yeah, this is as an alternative to nightlies. I will say, "To install the latest unreleased changes to the library"